samples/Compute/platform-image-policy/README.md

```diff
@@ -1,6 +1,6 @@
 # Only allow a certain VM platform image
 
-This policy ensures that only UbuntuServer, Canonical is allowed from the image repository
```

Comment:
This was one of the first samples to be added to the repo, before we had any naming conventions. Please update it to match the naming convention, e.g.

platform-image-policy => allowed-platform-images